download-build/action.yml

```diff
@@ -61,10 +64,16 @@ runs:
         JFROG_DL_BUILD: ${{ github.event.repository.name }}/${{ inputs.build-number }}
         JFROG_DL_EXCLUSIONS: ${{ inputs.exclusions }}
         JFROG_DL_REMOTE_REPO: ${{ inputs.remote-repo }}
-      run: jfrog rt download
-        --fail-no-op ${{ steps.define-download-options.outputs.jfrog_dl_options }}
-        --build "${JFROG_DL_BUILD}"
-        "${JFROG_DL_REMOTE_REPO}"
+        groupIdPath: ${{ inputs.filter }}
+      run: |
+        if [ -n "$groupIdPath" ]; then
+          JFROG_DL_REMOTE_REPO="${JFROG_DL_REMOTE_REPO}/$groupIdPath/"
+        fi
+        echo "$JFROG_DL_REMOTE_REPO"
+        jfrog rt download \
+          --fail-no-op ${{ steps.define-download-options.outputs.jfrog_dl_options }} \
+          --build "${JFROG_DL_BUILD}" \
+          "${JFROG_DL_REMOTE_REPO}"
```

Comment:
What about this?
```suggestion
      run: |
        jfrog rt download \
          --fail-no-op ${{ steps.define-download-options.outputs.jfrog_dl_options }} \
          --build "${JFROG_DL_BUILD}" \
          "${JFROG_DL_REMOTE_REPO}/$groupIdPath/"
```

Reply:
you end up with a double / at the end of the repo if groupIdPath is empty, I'm ok with verbosity